apps/frontend/src/main/mcp-server/start.sh

```
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+# MCP Server Launcher with Electron Mocking
+
+# Mock Electron before starting
+export NODE_OPTIONS="--require $(dirname "$0")/mock-electron.js"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)